### DISCUSSION TITLE
Is there a way to see speed test averages over a period of time (e.g., past day, week, month, etc.)?

### DISCUSSION THREAD

**Discussion by jonathanlaniado** [alexjustesen/speedtest-tracker#1830] [Q&A]
I haven't been able to find this functionality yet.

**Comment by alexjustesen** [comment]
Within the app, not currently but it's something coming after v1 which I'm trying to tag here in the next couple weeks. You can always export the data or integrate with InfluxDB to produce more charts/metrics.
